### ISSUE TITLE
Loadouts not saving properly in beta 2

### ISSUE THREAD

**Issue by AAraKKe** [kyleshay/DIM#73]
I have seen this happening before beta 2 but haven't had the time to comment. I create a load out lets say "Crota". Then I want to create a new one and the down menu has all the crota weapons still there but it has no name as a new loadout. I remove the "Crota" weapons and add the ones that I want in this loadout and save. Now I have two loadouts bu if I go to "Crota" it has the weapons I added to the new one I created and there is no loadout with "Crota" weapos, just to of them with the same weapons and different name.
In summary, it seems that you can create as many loadouts as you want but only one set of weapons is actually loaded in all loadouts.

**Comment by SunburnedGoose** [comment]
Related to #63.  This is fixed in the code, and will go out in Beta 3.  I'll push it today so you all can use it this weekend.
